### ISSUE TITLE
uninitialized constant HTTParty::HashConversions::ERB

### ISSUE THREAD

**Issue by ajsharp** [rainforestapp/rainforest-cli#26]
When I run the following command, I get this uninitialized constant error: `rainforest run all --token TOKEN --fg --fail-fast`.

Using gem version 1.0.1 on ruby 2.1.0.

**Comment by ian** [comment]
that's what you're running or thats the error?

**Comment by ajsharp** [comment]
Command: `rainforest run all --token TOKEN --fg --fail-fast`
Error: `uninitialized constant HTTParty::HashConversions::ERB`.

Backtrace:

```
/usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:33:in `normalize_param': uninitialized constant HTTParty::HashConversions::ERB (NameError)
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:29:in `block in normalize_param'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:29:in `map'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:29:in `normalize_param'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:15:in `block in to_params'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:15:in `each'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:15:in `map'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:15:in `to_params'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:148:in `normalize_query'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:129:in `body'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:158:in `setup_raw_request'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:98:in `perform'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty.rb:521:in `perform_request'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty.rb:475:in `post'
        from /usr/local/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty.rb:563:in `post'
        from /usr/local/lib/ruby/gems/2.1.0/gems/rainforest-cli-1.0.1/lib/rainforest/cli/http_client.rb:22:in `post'
        from /usr/local/lib/ruby/gems/2.1.0/gems/rainforest-cli-1.0.1/lib/rainforest/cli/runner.rb:22:in `run'
        from /usr/local/lib/ruby/gems/2.1.0/gems/rainforest-cli-1.0.1/lib/rainforest/cli.rb:25:in `start'
        from /usr/local/lib/ruby/gems/2.1.0/gems/rainforest-cli-1.0.1/bin/rainforest:7:in `<top (required)>'
        from /usr/local/bin/rainforest:23:in `load'
        from /usr/local/bin/rainforest:23:in `<main>'
```

**Comment by ian** [comment]
They're hardly ruby programmers. Is their gem broken?

**Comment by ajsharp** [comment]
This appears to be a httpparty issue: https://github.com/jnunemaker/httparty/pull/403

**Comment by ian** [comment]
Version or inherent problem

**Comment by tj** [comment]
:( older releases are busted as well

**Comment by ajsharp** [comment]
A lot of gems are affected by this. At this point, it either makes sense to re-release the rainforest-cli gem with a hard dependency on the last working version of httparty, or fork httparty and release it as a gem so you can depend on it in the rainforest-cli gemspec.

**Comment by ukd1** [comment]
@ajsharp sorry about this, if you want to submit a patch we'll happily merge, if not we can handle internally and notify you when it's fixed.

**Comment by ajsharp** [comment]
@ukd1 Submitted a PR at #27

**Comment by ukd1** [comment]
Closed by 667c2bb6c03681efd458ba9b77bb6daf2b3d4fdf

**Comment by ukd1** [comment]
@tj did the fix help?
